app/Dockerfile

```diff
@@ -2,8 +2,9 @@ FROM eclipse-temurin:17-jre-alpine
 
 # hadolint ignore=DL3018
 RUN apk update && \
-    apk --no-cache add curl redis sudo && \
-    apk upgrade openssl libssl3 libcrypto3 && \
+    apk --no-cache add curl redis sudo gcompat && \
+    apk search libgcrypt && \
```

Comment:
I'm not too experienced with `apk`, but is there a reason to run `apk search`?

Reply:
great catch!  I had added that while looking for options of what could be updated.  While libexpat had a direct package hit, the libgcrypt was harder to figure out, requiring the addition of gcompat in the lines above.  I'll remove the search as it shouldn't have been included to begin with.  Thanks Cory